File: Front/GAPInsurancesApp/src/app/models/customer.ts
export class CustomerModel {
    CustomerID: number;
    FullName: string;
    Address: string;
    Phone: string;
    Identification: string;

    CustomerModel() {
        this.CustomerID = -1;
        this.FullName = '';
        this.Address = '';
        this.Phone = '';
        this.Identification = '';
    }
}

File: Back/InsurancesAPI/Models/Business/Customer.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Models.Business
{
    public class Customer
    {
        [Key]
        public long CustomerID { get; set; }
        public string FullName { get; set; }
        public string Address { get; set; }
        public string Phone { get; set; }
        public string Identification { get; set; }        
    }
}
